packages/atomic/src/components/common/atomic-icon-button/atomic-icon-button.tsx

```diff
@@ -22,6 +22,7 @@ export class AtomicIconButton {
   @Prop({mutable: true}) public tooltip = '';
   @Prop({mutable: true}) public labelI18nKey!: string;
   @Prop({mutable: true}) public icon!: string;
+  @Prop({mutable: true}) public iconClass = 'w-4 h-4 shrink-0';
```

Comment:
Would it be better to add a part to the atomic-icon? 🤔

Reply:
Yes it's preferable we usually don't pass classes down to props